spring-cloud-sleuth-autoconfigure/src/main/java/org/springframework/cloud/sleuth/autoconfig/instrument/reactor/SleuthReactorProperties.java

```diff
@@ -89,9 +89,7 @@ public void setInstrumentationType(InstrumentationType instrumentationType) {
 		/**
 		 * Decorates on each operator, will be less performing, but logging will always
 		 * contain the tracing entries in each operator.
-		 * @deprecated to be removed in Sleuth 4.0.0
 		 */
-		@Deprecated
 		DECORATE_ON_EACH,
```

Comment:
I wonder about this one. Maybe adding information that this is maybe not deprecated but experimental is a good idea? Performance wise it's great to see such improvement however I'm pretty sure that there are edge cases where the context will not be properly propagated :grimacing: